examples/application_commands/message_command.py

```diff
@@ -0,0 +1,13 @@
+import nextcord
+from nextcord import Interaction
+
+client = nextcord.Client()
+
+
+@client.message_command(guild_ids=[...])  # limits guild's with this command
+async def my_message_command(interaction: Interaction, message):  
```

Comment:
```suggestion
async def my_message_command(interaction: Interaction, message: nextcord.Message):  
```
Typehinting should be added here for consistency.